Hmmm it looks like there is an iconv error:

```
  cargo:warning=In file included from libgit2/src/commit_list.c:12:
  cargo:warning=In file included from libgit2/src/odb.h:19:
  cargo:warning=In file included from libgit2/src/filter.h:12:
  cargo:warning=In file included from libgit2/src/attr_file.h:17:
  cargo:warning=In file included from libgit2/src/futils.h:14:
  cargo:warning=libgit2/src/path.h:430:10: fatal error: 'iconv.h' file not found
  cargo:warning=#include <iconv.h>
  cargo:warning=         ^~~~~~~~~
  cargo:warning=In file included from libgit2/src/merge_driver.c:12:
  cargo:warning=In file included from libgit2/src/merge.h:15:
  cargo:warning=In file included from libgit2/src/iterator.h:15:
  cargo:warning=In file included from libgit2/src/ignore.h:12:
  cargo:warning=In file included from libgit2/src/repository.h:24:
  cargo:warning=In file included from libgit2/src/attrcache.h:12:
  cargo:warning=In file included from libgit2/src/attr_file.h:17:
  cargo:warning=In file included from libgit2/src/futils.h:14:
  cargo:warning=libgit2/src/path.h:430:10: fatal error: 'iconv.h' file not found
  cargo:warning=#include <iconv.h>
  cargo:warning=         ^~~~~~~~~
  cargo:warning=1 error generated.
  exit code: 1
  ```

--- a/pkgs/development/tools/convco/default.nix
+++ b/pkgs/development/tools/convco/default.nix
@@ -1,0 +1,26 @@
+{ lib, rustPlatform, fetchFromGitHub, stdenv, openssl, perl, pkg-config, libiconv, Security }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "convco";
+  version = "0.3.2";
+
+  src = fetchFromGitHub {
+    owner = "convco";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0fqq6irbq1aikhhw08gc9kp0vbk2aminfbvwdlm58cvywyq91bn4";
+  };
+
+  cargoSha256 = "073sfv42fbl8rjm3dih1ghs9vq75mjshp66zdzdan2dmmrnw5m9z";
+
+  nativeBuildInputs = [ openssl perl pkg-config ];
+
+  buildInputs = lib.optionals stdenv.isDarwin [ libiconv Security ];
+
+  meta = with lib; {
+    description = "A Conventional commit cli";
+    homepage = "https://github.com/convco/convco";
+    license = with licenses; [ mit ];
+    maintainers = with maintainers; [ hoverbear ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -10692,6 +10692,10 @@ in
 
   crate2nix = callPackage ../development/tools/rust/crate2nix { };
 
+  convco = callPackage ../development/tools/convco {
+    inherit (darwin.apple_sdk.frameworks) Security;
+  };
+
   maturin = callPackage ../development/tools/rust/maturin { };
   inherit (rustPackages) rls;
   rustfmt = rustPackages.rustfmt;

